dkms_autoinstaller.in

```diff
@@ -23,21 +23,11 @@ elif [ -f /etc/rc.d/init.d/functions ]; then
     . /etc/rc.d/init.d/functions
 fi
 
-# We only have these functions on Debian/Ubuntu
-# so on other distros just stub them out
-if [ ! -f /etc/debian_version ]; then
-    alias log_daemon_msg='/bin/echo -n'
-    log_end_msg() { if [ "$1" = "0" ]; then echo " Done. "; else echo " Failed. "; fi; }
-    alias log_action_msg=/bin/echo
-fi
-
 exec="@SBINDIR@/dkms"
 prog=${exec##*/}
 
 test -f $exec || exit 0
 
-[ -e /etc/sysconfig/$prog ] && . /etc/sysconfig/$prog
```

Comment:
This would do more than just cleaning up the output, right? Wouldn't this break compatibility if someone was setting configuration in `/etc/sysconfig/dkms`? Although that's more of a RedHat convention.

Upon quick inspection I don't see variables actually being used here so maybe that's the reason for removing this line?

Reply:
Correct, but there has never ever been something installed or provided as `/etc/sysconfig/dkms` in the Red Hat packaging history. On top of that, everything is in `/etc/dkms`, so that inclusion is completely useless.